vault/audit.go

```diff
@@ -507,6 +513,11 @@ func (c *Core) removeAuditReloadFunc(entry *MountEntry) {
 
 // newAuditBackend is used to create and configure a new audit backend by name
 func (c *Core) newAuditBackend(ctx context.Context, entry *MountEntry, view logical.Storage, conf map[string]string) (audit.Backend, error) {
+	// Ensure that non-Enterprise versions aren't trying to supply Enterprise only options.
+	if hasInvalidAuditOptions(entry.Options) {
+		return nil, fmt.Errorf("enterprise-only options supplied")
```

Comment:
```suggestion
		return nil, fmt.Errorf("Enterprise-only options supplied")
```

Reply:
This is a nit, but I think we try and capitalize Enterprise in user facing stuff

Reply:
I took it from the logical system helpers error response we return when a feature is Enterprise only: 

>enterprise-only feature

e.g. in CE: `vault namespace create test`

Reply:
Aha! That's probably better then, thanks for clarifying!